web-access-app/src/main/java/edu/unc/lib/boxc/web/access/controllers/FullRecordController.java

```diff
@@ -273,12 +280,20 @@ public String handleUvViewerRequest(@PathVariable("pid") String pidString, Model
     private Map<String, Object> getViewerProperties(ContentObjectRecord briefObject, AccessGroupSet principals) {
         String viewerType = null;
         String viewerPid = null;
+        String streamingUrl = null;
+        String streamingType = null;
 
         boolean imageViewerNeeded = accessCopiesService.hasViewableFiles(briefObject, principals);
+        var workStreamingContent = accessCopiesService.getStreamingChildren(briefObject, principals);
+
         if (imageViewerNeeded) {
             viewerType = "uv";
         } else if (briefObject.getContentStatus().contains(FacetConstants.HAS_STREAMING)) {
             viewerType = "streaming";
```

Comment:
Maybe we should also set streamingUrl and streamingType in viewerProperties for file records that have streaming, so its always accessible from the same place in the response? Might not need getProperty in streamingPlayer.vue then.